src/core_tests.rs

```diff
@@ -38,6 +38,9 @@ use std::sync::mpsc;
 use std::thread;
 use types::MessageId;
 use xor_name::XorName;
+use maidsafe_utilities;
+// Add the following line to tests to enable logging:
```

Comment:
I don't think these two lines should be included - their point is as relevant here as in _any_ source file.